src/specs/api.v2.yaml

```diff
@@ -531,6 +567,8 @@ components:
       $ref: './models/experiment-bodies/GetAllExperiments.v2.yaml'
     CreateSample:
       $ref: './models/samples-bodies/CreateSample.v2.yaml'
+    GetSamples:
+      $ref: './models/samples-bodies/Samples.v2.yaml'
```

Comment:
There's some alternatives here, you could:

- rename `Samples.v2.yaml` to `GetSamples.v2.yaml` and merge the 2 validation files or
- Get rid of `GetSamples.v2.yaml` and in this line write:
```
type: array
items:
  $ref: ./Sample.v2.yaml
```

Reply:
I personally like more the first option (`GetSamples.v2.yaml`), but let me know what you think

Reply:
We might want to have the individual sample spec for other endpoints, I just made it the second way it is simple enough to not be unconvenient I think.

Reply:
It is a good rule of thumb to make things easy to modify but in the way that best fits our current usage, not planning for cases in the future that might need it without knowing whether they will ever come up.

Based on this I would still recommend doing one fo the 2 options (or another one you come up with that you think fits better)